ResumeParseAndNormalize.md

```diff
@@ -348,6 +349,12 @@ Response Structure
 }
 ```
 
+PII Scrubbing
+-----------
+The service makes a best attempt at scrubbing Personally Identifiable Information (i.e. Social Security Numbers and Driver's License Numbers). PII scrubbing logic is applied to the `raw_resume_text`, `clean_resume_text`, and `resume_html` fields only. Anything found to resemble a SSN or Driver's License Number in these fields will be replaced with either: "***Social Security Number Removed***" or "***Driver's License Number Removed***". Parsed data fields (such as the phone number fields) will therefore be unaffected by false-positive PII removals.
```

Comment:
sorry, one last thing I just discovered by previewing this (btw, you should always preview your changes in markdown files). you need to escape your asterisks; currently it's bold+italicizing these strings instead of showing the asterisks. it's important because people might write code that expects these exact string literals. you might want to demonstrate the DOB Removed string in the next paragraph too.